doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
 DOOMGRADER_ROOT=~/doomgrader
 STEAM_PATH=~/.steam/steam/steamapps/common
 
 # internal paths
 DOWNLOAD_PATH=$DOOMGRADER_ROOT/files
 DEPOTDOWNLOADER_PATH=$DOOMGRADER_ROOT/depotdownloader
 
-# prompt for steam credentials
+# function for copying files with prompt
+DOOMGRADER_COPY()
+{
+# copy game files to steam dir
+echo "Copying the game from $DOWNLOAD_PATH to $STEAM_PATH/DOOMEternal/"
+echo "THIS WILL OVERWRITE YOUR EXISTING DOOM ETERNAL GAME FILES!"
```

Comment:
Again, I don't think we need all caps and exclamation marks in the code. The wording is good though!

Reply:
Can we leave overwrite captalized? I don't want users to easily ignore the text.

Reply:
Sure